src/main/java/games/strategy/engine/lobby/client/ui/LobbyFrame.java

```diff
@@ -202,14 +191,9 @@ private void showServerMessage(final LobbyServerProperties props) {
     }));
     rVal.add(SwingAction.of("Mute Player", e -> {
       final List<String> muteTypes = new ArrayList<>();
-      // too many choices is confusing our mods, just give them 3 choices: name, everything, or cancel.
-      // muteTypes.add("IP Address");
-      // muteTypes.add("Mac Address");
+      muteTypes.add("Mac Address");
       muteTypes.add("Username only");
-      // muteTypes.add("IP, Mac");
-      // muteTypes.add("Name, IP");
-      // muteTypes.add("Name, Mac");
-      muteTypes.add("Name, IP, Mac");
+      muteTypes.add("Name + Mac");
```

Comment:
Match criteria is a case insenstive contains:
```
        if (selectedMuteType.toLowerCase().contains("name")) {
          controller.muteUsername(clickedOn, null);
        }
```
Consolidated the two lists to one: 8482995